GERMLINE_0001.cpp

```diff
@@ -78,7 +119,9 @@ int main(int argc, char* argv[])
 		<< '\t' << "-homoz-only" << '\t' << "Look for autozygous/homozygous segments only, does not detect IBD" << endl
 		<< '\t' << "-haploid" << '\t' << "Treat input individual as two fully phased chromosomes with no recombination\n\t\toutput IDs with 0/1 suffix for chromosome destinction" << endl
 		<< '\t' << "-h_extend" << '\t' << "Extend from seeds if *haplotypes* match" << endl
-		<< '\t' << "-w_extend" << '\t' << "Extend, one marker at a time, beyong the boundaries of a found match" << endl;
+		<< '\t' << "-w_extend" << '\t' << "Extend, one marker at a time, beyong the boundaries of a found match" << endl
+		<< '\t' << "-old_samples" << '\t' << "List of old individuals to cross compare." << endl
+		<< '\t' << "-new_samples" << '\t' << "List of new individuals." << endl;
```

Comment:
With this parameterization, will new_samples be automatically compared to each other? Or does the caller need to include those IDs in new_samples.txt and old_samples.txt?

Reply:
At this stage they are not, this is sligthly more flexible (you do two calls, but you have the options)

Reply:
Ok, makes sense. I think we should rename the -old_samples flag, then to `-samples_to_compare_to` or something like that -- it's not obvious that you can include a new_sample in old_samples in order to get it compared to itself (or other new samples)